PraceDomowe/PracaDomowa7+8/Nahorny_Jakub/PracaDomowa78.ipynb

```diff
@@ -0,0 +1,4729 @@
+{
```

Comment:
<p>To chyba powinien być wykres słupkowy. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>